shop-info-service/src/functions/getProductById/handler.ts

```diff
@@ -0,0 +1,33 @@
+import { PRODUCTS } from "../../mocks/products";
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from "aws-lambda";
+import { emitConnectionDelay } from "../../mocks/delayFunc";
+
+const NOT_FOUND: string = 'Not Found Product With Id';
+
+export const getProductById = async (event: APIGatewayProxyEvent): Promise<APIGatewayProxyResult> => {
+  const id = event.queryStringParameters?.id;
+
+  try {
+    await emitConnectionDelay(500);
+    const product = PRODUCTS.find(x => x.id === id);
+
+    if (!product) {
+      throw `${NOT_FOUND} ${id}`;
```

Comment:
throw error model here

Reply:
Sure I can
But the is no such info in the task, only this one
"+1 (All languages) - Main error scenarios are handled by API ("Product not found" error)."

Reply:
updated